docs/community/opensource.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I would name the file `open-source.md` so that the URL becomes `/open-source` and not `/opensource` but looks good otherwise!